packages/bindings-test/src/multitest.rs

```diff
@@ -542,4 +558,6 @@ mod tests {
             .unwrap();
         assert_eq!(end, mintable);
     }
+
+    // TODO: Delegate / Undelegate tests
```

Comment:
Consider creating an issue for that?

Reply:
Issue: https://github.com/confio/poe-contracts/issues/126.